web/lua/opmserver/utils.lua

```diff
@@ -51,4 +63,71 @@ function _M.read_file(path)
 end
 
 
+function _M.gen_random_string(length)
+    length = length or 32
+    length = length / 2
+    local r = random_bytes(length, true)
```

Comment:
need to call `math.randomseed` in `init_by_lua` phase for a 'real' random string here.